### PR TITLE
v2.24.0 – Updating browser support checklist

### PR DESCRIPTION
**Pull request by ashleynolan**
### Changed
- Browser Support documentation updated

### CODE REVIEW

File: docs/src/templates/pages/documentation/general/browser-support.hbs [hunk 1]
```diff
@@ -11,7 +11,7 @@ navactive: browser-support
 
 The number of browsers available to our users is vast, especially for those using mobile devices. Below, we have defined a rating system that helps us to specify the level of browser support we aim to achieve and to clarify what level of testing is required.
 
-As with any browser support checklist, this list should be reviewed often and updated in-line with the browsers and devices that our users are choosing to access our platforms via.
+As with any browser support check-list, this list should be reviewed often and updated in-line with the browsers and devices that our users are choosing to access our platforms via.
```
Comment (xander-marjoram) [inline docs/src/templates/pages/documentation/general/browser-support.hbs]: `checklist` is correct

File: package.json [hunk 2]
```diff
@@ -1,69 +1,66 @@
...
-  },
-  "browserslist": [
-    "> 5%",
-    "last 2 versions",
-    "Safari >= 8",
-    "not ie 10"
-  ]
+    "name": "je-global-component-library",
+    "title": "Just Eat – Component Library",
+    "description": "Component Library and Guidelines for all of Just Eats Front-End Platforms",
```
Comment (xander-marjoram) [inline package.json]: `Eat's`

Comment (ashleynolan) [reply package.json]: Fixed

File: docs/src/templates/pages/documentation/general/browser-support.hbs (hunk 1 above)
Comment (ashleynolan) [reply docs/src/templates/pages/documentation/general/browser-support.hbs]: Fixed

**Pull request merged**